coc-powershell.build.ps1

```diff
@@ -0,0 +1,78 @@
+#
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+param(
+    [string]$EditorServicesRepoPath = $null
+)
+
+#Requires -Modules @{ModuleName="InvokeBuild";ModuleVersion="3.0.0"}
+
+# Grab package.json data which is used throughout the build.
```

Comment:
It looks like we don't use these variables anywhere except to log... The logging is still useful, but maybe change this comment